libcxx/test/std/library/description/conventions/customization.point.object/cpo.compile.pass.cpp

```diff
@@ -77,23 +79,41 @@ static_assert(test(std::ranges::ssize, a));
 
 // [range.factories]
 // views::empty<T> is not a CPO
-static_assert(test(std::views::iota, 1));
 static_assert(test(std::views::iota, 1, 10));
-//static_assert(test(std::views::istream<int>, 1);
+static_assert(test(std::views::iota, 1));
+static_assert(test(std::views::istream<int>, stream));
+static_assert(test(std::views::repeat, 1));
```

Comment:
The followings need to be guarded with `TEST_STD_VER >= 23`:
- `repeat`
- `adjacent_transform`
- `adjacent`
- `as_const`
- `as_rvalue`
- `cartesian_product`
- `chunk`
- `chunk_by`
- `enumerate`
- `join_with`
- `stride`
- `zip_transform`
- `zip`

The followings need to be guarded with `TEST_STD_VER >= 26` (although currently none of them is implemented):
- `cache_latest`
- `concat`
- `to_input`

Reply:
I think I have done it. I noticed then missing tests and I was planning to submit a patch too.